demos/guide/README.md

```diff
@@ -5,11 +5,73 @@ The demos in this directory are the complete source code for the applications
 developed in the OpenSSL Guide tutorials. Refer to the various tutorial pages in
 the [guide] for an extensive discussion on the demos available here.
 
+They must be built before they can be run. An example UNIX style Makefile is
+supplied. Just type "make" from this directory on a Linux/UNIX system.
+
+Running the TLS Demos
+---------------------
+
 To run the demos when linked with a shared library (default) ensure that
 libcrypto and libssl are on the library path. For example, assuming you have
 already built OpenSSL from this source and in the default location then to run
 the tls-client-block demo do this:
 
-LD_LIBRARY_PATH=../.. ./tls-client-block
+LD_LIBRARY_PATH=../.. ./tls-client-block hostname port
```

Comment:
Shouldn't we be recommending util/wrap.pl for this?

Reply:
I'm not sure that we should. `util/wrap.p` sets up a whole load of other environment variables (e.g. `OPENSSL_CONF`, `OPENSSL_CONF_INCLUDE` etc) that are specifically intended for us during our own testing. I see `util/wrap.pl` as something mostly internal. I'm not sure we want to be encouraging end users to use it?